low-level/json_msgs/messages/base_msg.py

```diff
@@ -39,6 +45,30 @@ def __init__(self):
     def getJson(self):
         raise NotImplementedError("Subclasses should implement this!")
 
+    def prepare_message(self, jsonMsg, message_type):

```

Comment:
A Docstring can be added for `prepare_message` function.

Reply:
Added.